neqo-transport/src/connection.rs

```diff
@@ -551,6 +563,8 @@ impl Connection {
         }
 
         self.tps.borrow_mut().remote_0rtt = Some(tp);
+        self.loss_recovery
+            .set_latest_rtt(Duration::from_millis(smoothed_rtt));
```

Comment:
Given that 0 means that we don't have a smoothed RTT, it's probably best to do this:
```suggestion
        if smoothed_rtt > GRANULARITY {
          self.loss_recovery
              .set_latest_rtt(Duration::from_millis(smoothed_rtt));
        }
```
Note that this uses the timer granularity that we assume.

(Separately, we need to make that granularity 1ms.  You can avoid having to worry about that factoring in here by making the tests use an RTT estimate greater than 20ms for the moment.)

Reply:
I put GRANULARITY as a constant at the top of the file then.